pages/validators/preparing_for_genesis.md

```diff
@@ -62,34 +51,32 @@ dydxprotocold version --long
 The output should look like this (**make sure** the `version` and `commit` is consistent):
 
 ```bash
-commit: e2ea4e4ebebbe708f42d77c890214941fc2830ae
+commit: TODO
```

Comment:
Since the testnet #4 binary isn't ready until Thursday, should we explicitly point validators to last stable version on testnet #3? Otherwise, we won't update this `TODO` item by the time validators see the ` Preparing for Genesis` doc.

Reply:
done!